src/main/java/org/zalando/nakadi/repository/db/SchemaRepository.java

```diff
@@ -37,12 +37,7 @@ public EventTypeSchema getSchemaVersion(final String name, final String version)
                 "WHERE ets_event_type_name = ? AND ets_schema_object ->> 'version' = ?";
 
         try {
-            final List<EventTypeSchema> schemas =
-                    jdbcTemplate.query(sql, new Object[]{name, version}, new SchemaRowMapper());
-            if (schemas.size() != 1)
-                throw new InternalNakadiException(
```

Comment:
you do not need InternalNakadiException  on line https://github.com/zalando/nakadi/pull/505/files#diff-96bb991c5b6b4a062777cd0554da6b32L35